packages/x-components/src/x-modules/popular-searches/components/__tests__/popular-searches.spec.ts

```diff
@@ -27,15 +27,15 @@ describe('testing popular searches component', () => {
   });
 
   it('is an XComponent', () => {
-    expect(isXComponent(popularSearchesWrapper.vm)).toEqual(true);
+    expect(isXComponent(popularSearchesWrapper.vm)).toBe(true);
```

Comment:
These changes were done by you or by the linter? just to know

Reply:
That change was made by the linter, F

Reply:
We might have to disable that rule as well